smithy-aws-protocol-tests/model/rpcV2/cbor-structs.smithy

```diff
@@ -319,4 +404,167 @@ structure SimpleScalarStructure {
     longValue: Long,
     shortValue: Short,
     stringValue: String,
-}
\ No newline at end of file
+}
+
+
+apply DefaultFieldInputOutput @httpRequestTests([
+    {
+        id: "RpcV2CborEmptyInputShouldSerializeDefaultValues",
+        documentation: "Serializes an empty input with default values filled in.",
+        protocol: rpcv2Cbor,
+        method: "POST",
+        uri: "/service/RpcV2Protocol/operation/DefaultFieldInputOutput",
+        // http://ec2-54-84-9-83.compute-1.amazonaws.com/hex?value=qWlieXRlVmFsdWUFa2RvdWJsZVZhbHVl%2Bz%2F%2BOVgQYk3TcWZhbHNlQm9vbGVhblZhbHVl9GpmbG9hdFZhbHVl%2BkDz989saW50ZWdlclZhbHVlGQEAaWxvbmdWYWx1ZRkmkWpzaG9ydFZhbHVlGSaqa3N0cmluZ1ZhbHVlZnNpbXBsZXB0cnVlQm9vbGVhblZhbHVl9Q%3D%3D        body: "qWlieXRlVmFsdWUFa2RvdWJsZVZhbHVl+z/+OVgQYk3TcWZhbHNlQm9vbGVhblZhbHVl9GpmbG9hdFZhbHVl+kDz989saW50ZWdlclZhbHVlGQEAaWxvbmdWYWx1ZRkmkWpzaG9ydFZhbHVlGSaqa3N0cmluZ1ZhbHVlZnNpbXBsZXB0cnVlQm9vbGVhblZhbHVl9Q=="
```

Comment:
`body` appears to be commented out accidentally.

Reply:
Thanks. Have fixed it now.